camelot/view/action_steps/item_view.py

```diff
@@ -251,16 +275,21 @@ def render(cls, gui_context, step, engine):
 
         # create QML item
         view = create_qml_item(
-            QtCore.QUrl("qrc:/vortex/qml/common/TablePage.qml"),
+            #QtCore.QUrl("qrc:/Vortex/qml/common/TablePage.qml"),
+            QtCore.QUrl("qrc:/Vortex/TablePage.qml"),
```

Comment:
ik zou dus verwachten van hier de 'backend' mee te geven, maar dat is in deze fase waarschijnlijk te veel gedoe ivm de constructie vh object ...

Reply:
Klopt